File: src/Player.js
// src/Player.js
import React from 'react';

const Player = ({ name, piece, balance, properties }) => {
  return (
    <div className="player-container">
      <h3>{name}</h3>
      <p>Piece: {piece}</p>
      <p>Balance: ${balance.toFixed(2)}</p>
      <p>Properties: {properties.length}</p>
    </div>
  );
};

export default Player;

File: backend/app/schemas/player.py
from pydantic import BaseModel
from typing import List, Optional
from .property import Property

class PlayerBase(BaseModel):
    name: str
    piece: str
    balance: float = 15000000.0
    position: int = 0
    side: str = "bottom"

class PlayerCreate(PlayerBase):
    pass

class Player(PlayerBase):
    id: int
    game_id: int
    properties: List[Property] = []

    class Config:
        orm_mode = True